src/main/java/org/sciborgs1155/robot/Robot.java

```diff
@@ -218,23 +218,27 @@ private void configureBindings() {
         .povUp()
         .whileTrue(shooter.runShooter(-ShooterConstants.IDLE_VELOCITY.in(RadiansPerSecond)));
 
+    // driver intake button
+    driver.leftTrigger().whileTrue(intake.intake());
+
     // intake (right trigger / top left bump)
     operator
-        .leftBumper()
+        .leftTrigger()
         .whileTrue(intake.intake().deadlineWith(feeder.slowForward()))
         .whileTrue(led.rainbow());
 
-    // operator feed (left trigger)
+    // operator feed (left trigger) underfeeding
     operator
-        .leftTrigger()
+        .leftBumper()
         .whileTrue(
-            shooting.shootWithPivot(PivotConstants.FEED_ANGLE, ShooterConstants.DEFAULT_VELOCITY));
+            shooting.shootWithPivot(
+                PivotConstants.UNDERFEED_ANGLE, ShooterConstants.DEFAULT_VELOCITY));
 
-    // shoot 45 deg FAST (far as possible)
-    // operator
-    //     .povLeft()
-    //     .whileTrue(
-    //         shooting.shootWithPivot(Radians.of(0.619), ShooterConstants.MAX_VELOCITY));
+    // shoot 45 deg FAST (far as possible) overfeeding
```

Comment:
This is dangerous, untested, and was made as a joke. Please comment this out or remove this entirely until a proper overfeed is added and tested 😭😭